src/components/widget/Lookup/Lookup.js

```diff
@@ -160,10 +161,18 @@ class Lookup extends Component {
         }
     }
 
+
     generatingPropsSelection = () => {
         const {onChange} = this.props;
         const {properts} = this.state;
         const propertiesKeys = Object.keys(properts);
+        let isClose = true;
+
+        this.props.defaultValue.map (item => {
+            if(!item.value){
```

Comment:
this whole mapping, isn't it just boolean value of length of filtered array with !item.value?